### DISCUSSION TITLE
Thank You

### DISCUSSION THREAD

**Discussion by nq-gh-nerd** [jbruchon/elks#2061] [General]
I created a user account just to post this.  I last worked with ELKS over 20 years ago when it was v0.0.83 and then spent much of the intervening time doing things like raising children.

Now that I have time to look at this IBM PC again, I'm ecstatic to find that ELKS is alive and well in 2024.

Thank all of you so much for this.  I can't even begin to tell you.

**Comment by ghaerr** [comment]
Hello @nq-gh-nerd,

Wow v0.0.83 was a long long time ago. Thank you for your post, and let us know how ELKS is working for you. There has been a lot of work put into ELKS in the last five years, and it seems to run pretty well now.

Thank you!
